enf-reducer/cut-unnecessary-or.metta

```diff
@@ -0,0 +1,19 @@
+ ; Cut unnecessary OR, DOESN'T process sub expressions
+ ; To be used with iterator
```

Comment:
Can you give a bit more information than that if you can. If you can't just let me know and we will add it later :). The comment should ideally be what the function does, possibly with example input output to give the reader a glance of what it does without the need to look into the code :)